deploy/docker/Dockerfile

```diff
@@ -29,18 +29,25 @@ COPY --from=builder /emqx-rel/emqx /opt/emqx
 
 RUN ln -s /opt/emqx/bin/* /usr/local/bin/
 
-RUN apt-get update; \
-    apt-get install -y --no-install-recommends ca-certificates procps; \
+RUN groupadd -r -g 1000 emqx && \
```

Comment:
I think it's better to group all operations in one RUN block, otherwise Docker will create two separate FS layers.